module_generator/cookiecutter-template/{{cookiecutter.module_name}}/README.md

```diff
@@ -0,0 +1,5 @@
+# {{cookiecutter.module_name}} Module
```

Comment:
Can we use the recommended README from the docs here? https://docs.viam.com/how-tos/create-module/#optional-create-a-readme

Reply:
Oh I didn't even realize there was a template! Thank you